### ISSUE TITLE
is there a keyboard shortcut for prettify graphQl

### ISSUE THREAD

**Issue by Maxhodges** [getinsomnia/insomnia#2211]
![image](https://user-images.githubusercontent.com/957271/82735731-c83aba00-9d5e-11ea-8d6a-885e7424d39e.png)

can i run this with a keyboard command?

**Comment by fvieira** [comment]
This would be very useful, and it doesn't seem to exist yet.

**Comment by mheob** [comment]
Yes, there is a shortcut. It is called "Beautify Active Code Editors".
The default Value on a Mac is <kbd>SHIFT</kbd>+<kbd>CMD</kbd>+<kbd>I</kbd>.

**Comment by italosantana** [comment]
does anyone know on linux?
